docs/core/misc/boot.md

```diff
@@ -139,8 +139,16 @@ Vendor trust is stored as bitmap where unset bit means the feature is active.
 |  2  | 0x0004 | wait 4 seconds                          |
 |  3  | 0x0008 | wait 8 seconds                          |
 |  4  | 0x0010 | use red background instead of black one |
-|  5  | 0x0020 | require user click                      |
+|  5  | 0x0020 | require user click to continue          |
 |  6  | 0x0040 | show vendor string (not just the logo)  |
+|  7  | 0x0080 | allow access to pairing secret          |
+|  8  | 0x0100 | disable access to pairing secret        |
+
+Bits 0 to 6 represent vendor screen settings. The wait times are additive.
+
+Two bits are used for access to the pairing secret for historical reasons.
+Bit 7 is used in T2B1 and newer models.
```

Comment:
Currently bit 7 is used in T2B1 and newer models:
https://github.com/trezor/trezor-firmware/blob/39e1085295b9ebeafcdfdcd3c5cbce6891c77a0a/python/src/trezorlib/firmware/vendor.py#L63-L64

I think this is the right solution:
1. Since we are already taking up two bits, we might as well use them. Redundancy is good for security.
2. We can test whether access to the secret is allowed by defining `VTRUST_SECRET_ALLOW = 0x0100` for all models and testing `(vhdr.vtrust & VTRUST_SECRET_MASK) == VTRUST_SECRET_ALLOW`. Otherwise we would have to define `VTRUST_SECRET_ALLOW` differently for newer models. Less ifdefs and differences in meaning is always better IMHO.

Reply:
okay, yes, agreed, but the wording used makes it look like this is a typo, and it doesn't fully explain the situation either.

alternative proposal:
```
On T2B1 only bit 7 is evaluated.
On T3T1 and newer models, both bits 7 and 8 are evaluated.
```

Reply:
https://github.com/trezor/trezor-firmware/pull/3741/commits/22caf2d333767c50c241e4b998c4ba1305cb209c

I used simply "on newer models", to avoid ambiguity about T3B1.